vnext/src/Libraries/Components/AccessibilityInfo/AccessibilityInfo.windesktop.js

```diff
@@ -4,59 +4,159 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * Portions copyright for react-native-windows:
- *
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License.
  * @format
  * @flow
  */
+
 'use strict';
 
-var RCTDeviceEventEmitter = require('../../EventEmitter/RCTDeviceEventEmitter');
+const RCTDeviceEventEmitter = require('../../EventEmitter/RCTDeviceEventEmitter');
+const UIManager = require('../../ReactNative/UIManager');
 
-var TOUCH_EXPLORATION_EVENT = 'touchExplorationDidChange';
+import NativeAccessibilityInfo from './NativeAccessibilityInfo';
```

Comment:
>NativeAccessibilityInfo [](start = 7, length = 23)

This is now included in a flow-strict file, meaning our typings needed to match. The Android variant of this (and I think iOS) will guard against a missing native module already, so I picked that up verbatim and logged an item to try to share this between platforms.